guides/common/modules/con_postgresql-as-an-external-database-considerations.adoc

```diff
@@ -21,3 +21,12 @@ If you suspect that the PostgreSQL database on your {Project} is causing perform
 Queries that take longer than one second are typically caused by performance issues with large installations, and moving to an external database might not help.
 If you have slow queries, contact Red Hat Support.
 endif::[]
+
+.Next steps
+
+If you want your {Project} deployment to use an external PostgreSQL database, do one of the following:
+
+* Change the database setting when installing {Project}. For details, see {InstallingServerDocURL}configuring-dns-dhcp-and-tftp_{project-context}Configuring_Installation_{project-context}[Configuring {Project} installation]
```

Comment:
```suggestion
If you want your {Project} deployment to use an external PostgreSQL database, do one of the following:

* Change the database setting when installing {Project}.
For more information, see {InstallingServerDocURL}configuring-dns-dhcp-and-tftp_{project-context}Configuring_Installation_{project-context}[Configuring {Project} installation] in _{InstallingServerDocTitle}_.
```

suggestion: split line by sentence; use "For more information"; add name of the guide in italic; end with dot. All to align to our current conventions/style/pattern.

Just FYI: For Every XyzDocURL, there's a XyzDocTitle attribute. General rule: if we link to another guide, we mention the guide. We don't do this when we use  `xref:`.